.github/workflows/pipeline.yml

```diff
@@ -18,6 +18,14 @@ jobs:
     - run: docker build --target test --tag todo-app:test .
     - run: docker run --name tasko-test todo-app:test
 
+  scan:
+    name: Scan
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: docker build --target test --tag todo-app:test .
+    - run: docker run --entrypoint poetry todo-app:test run safety check
+
   docker-build-and-deploy:
     name: Build & deploy production docker image
     needs: test
```

Comment:
Worth considering - would you like this to  block on vulnerabilities as well?

Definitely arguments in both directs - a minor warning might be something we regularly want to recognise without blocking, but equally might mean we stop paying attention entirely to the scanner